lib/standalone.mjs

```diff
@@ -116,6 +100,20 @@ export default async function standalone({
 		quietDeprecationWarnings,
 	});
 
+	/** @type {Formatter} */
+	let formatterFunction;
+
+	/** @see https://github.com/stylelint/stylelint/issues/7447 */
+	if (!quietDeprecationWarnings && formatter === 'github') {
+		emitDeprecationWarning(
+			'"github" formatter is deprecated.',
+			'GITHUB_FORMATTER',
+			'Please use "stylelint-actions-formatters" instead.',
+		);
+	}
+
+	formatterFunction = await getFormatter(stylelint, { formatterType: 'json' });
```

Comment:
[suggestion] I think calling `getFormatter()` once is enough and `formatterType: 'json'` is no longer needed:

```suggestion
	const formatterFunction = await getFormatter(stylelint);
```

Reply:
Well, one seems like it's enough indeed. Pretty sure at one point it wasn't though. Hehe. To be honest, I'm not sure why or how it works now, by being called only once before... Is it because we removed the getConfigFromFile logic?

Reply:
> Is it because we removed the getConfigFromFile path logic?

Right. In Stylelint (internally using [Cosmiconfig](https://github.com/cosmiconfig/cosmiconfig)), there can be multiple config files in a project, e.g.

```
(root)
├── src/
│   ├── a.css 
│   └── .stylelintrc.json
├── b.css 
└── .stylelintrc.json
```

So, which configuration file to use depends on the source file that is currently being analyzed. That's why `getConfigFromFile()` is used.

However, the `formatter` option makes sense only at the root config file. So, `getConfigFromFile()` is not required in this case.